### DISCUSSION TITLE
Internal links

### DISCUSSION THREAD

**Discussion by bartart3d** [vkbo/novelWriter#1999] [Ideas]
Hello,

I am almost at the 60000 word count of my first writing project which is progressing very slowly because of lack of time. 
Because of the slow progress, details get lost in my mind. That's why I try to keep them stored in the notes. That's it purpose of course, but now I'm starting to feel the need for looking up in which scene I decided to add that detail.

Here a semi-fictional example, I have this character, John, who has brown hair, brown eyes, who is tall, and very thin. 
Half a year ago, in a particular scene I added a certain detail, for example that he has a x-shaped scar on his arm.
In the scene I also describe the circumstances where that scar came to be.  In my character note, I write down that detail. In other words, my characters grow in detail as I write and I add detail to the character note.

Now half a year later I want to use that detail again, that scar, only, I am twenty or thirty scenes further, I remember where that scar came from, but I don't remember why I needed it in the story in the first place. So I want to look up that scene, which is of course, quite the search.

For that reason, creating a way to link back, from the note to the scene could come in very handy!

grtz,
Bart

**Comment by vkbo** [comment]
Yes, there is indeed a plan for something like that. Depending on the case, you could use any of these features:

* Using the `@mention` keyword in #1822, which is useful for stating that a person or place has been mentioned, referenced, or described without being a part of the scene.
* You could also reference the scene directly form the note using the feature planned for #1920, where also novel documents can have tags and a new keyword.
* Or you could add a note comment like described in #1133, which would also be indexed.
